stdlib/pprint.pyi

```diff
@@ -110,3 +111,41 @@ class PrettyPrinter:
     def isreadable(self, object: object) -> bool: ...
     def isrecursive(self, object: object) -> bool: ...
     def format(self, object: object, context: dict[int, int], maxlevels: int, level: int) -> tuple[str, bool, bool]: ...
+    def _format(
+        self, object: object, stream: IO[str], indent: int, allowance: int, context: dict[int, int], level: int
+    ) -> None: ...
```

Comment:
`IO` is generally not recommended to use in argument annotations. From looking at the code, only `write()` seems to be called on the stream, so `SupportsWrite` (from `_typeshed`) seems like a suitable replacement.

Affects the methods below as well.

Reply:
There's no need to change old code in this PR, although it would surely be appreciated.